tests/assets/nav/_includes/main.vto

```diff
@@ -44,4 +44,4 @@
       {{ /if }}
     </nav>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
This the cause of the failing tests (a new empty line).
Can you run `deno task test tests/nav.test.ts -- --update` to update them?

Reply:
done